src/AppBundle/Controller/DashboardController.php

```diff
@@ -8,10 +8,23 @@
 
 class DashboardController extends Controller
 {
+
+    /**
+     * @Route("/", name="index")
+     */
+    public function indexAction(){
+        $user = $this->getUser();
+        if($user) {
+            return $this->redirectToRoute('dashboard');
+        }
+        else {
+            return $this->redirectToRoute('login');
+        }
+    }
     /**
-     * @Route("/", name="dashboard")
+     * @Route("/secured/dashboard", name="dashboard")
```

Comment:
Same here `secured` => `protected`

Reply:
Secured changed to protected, as you've requested, in all the routes and firewall rule.